### ISSUE TITLE
Object.assign breaks when used with lerna

### ISSUE THREAD

**Issue by tylerfowle** [kevva/dir-glob#12]
https://github.com/kevva/dir-glob/blob/4fc6c6e4a821a64dfd46205bc6c665082e82b022/index.js#L41

TypeError: Expected `cwd` to be of type `string` but received type `undefined`

I'm running into this error while using postcss-cli with lerna to run an autoprefixer tasks.
`"autoprefixer": "../../node_modules/.bin/postcss --use autoprefixer -r css/stylesheet.css --scripts-prepend-node-path"`

ive changed this:
        `opts = Object.assign({cwd: process.cwd()}, opts);`

to this:
	`opts.cwd = process.cwd();`

as a work around.

full error:
```lerna ERR! yarn run dev stderr:
npm WARN lifecycle The node binary used for scripts is /var/folders/pt/w9xns7l93zxbg9d9lxrx2jsrvbj7pq/T/yarn--1547151123573-0.09679202688283062/node but npm is using /usr/local/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.
TypeError: Expected `cwd` to be of type `string` but received type `undefined`
    at module.exports (/Users/..../node_modules/dir-glob/index.js:52:25)
    at globDirs (/Users/..../node_modules/globby/index.js:58:10)
    at getPattern (/Users/..../node_modules/globby/index.js:61:64)
    at Promise.all.globTasks.map.task (/Users/..../node_modules/globby/index.js:72:70)
    at Array.map (<anonymous>)
    at module.exports (/Users/..../node_modules/globby/index.js:72:42)
    at Promise.resolve.then (/Users/..../node_modules/postcss-cli/index.js:58:14)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
    at Function.Module.runMain (module.js:696:11)```

**Comment by dhruv-m-patel** [comment]
I get the exact same error as this, but I'm not using lerna.

Noticed that [there was a PR](https://github.com/kevva/dir-glob/pull/9) that was closed in favor of [a change in globby](https://github.com/sindresorhus/globby/pull/99) which I don't think is reflecting yet and is causing this issue.

**Comment by kevva** [comment]
Duplicate of #10.

@tylerfowle, see https://github.com/postcss/postcss-cli/issues/265.

@dhruv-m-patel, it should be fixed in `globby@8.0.2`.
